src/libcore/lib.rs

```diff
@@ -150,7 +151,13 @@ pub mod iter;
 pub mod option;
 pub mod raw;
 pub mod result;
+
+#[cfg(stage0)]
+#[path = "simd_old.rs"]
```

Comment:
you can possible replace the double-mod with just `#[cfg_attr(stage0, path = "simd_old.rs")] pub mod simd;`